dependencies/Dockerfile

```
@@ -1,26 +1,62 @@
+COPY --chmod=0755 build_libheif.sh .
+RUN --mount=type=secret,id=github_token \
+    && export GITHUB_TOKEN=$(cat /run/secrets/github_token) \
+    && export $(cat /env) && export $(cat /versions.env) && ./build_libheif.sh
 
-RUN mkdir -p /tmp/build
-WORKDIR /tmp/build
+FROM base-prep AS build-imagemagick
+ARG IMAGEMAGICK_VERSION
+WORKDIR /tmp/build-imagemagick
+COPY --chmod=0755 build_imagemagick.sh .
+RUN --mount=type=secret,id=github_token \
```

Hadolint (2.12.0)

[error] Unexpected start of line. If breaking lines, |/||/&& should be at the end of the previous one.

(SC1133)